Formula/luarocks.rb

```diff
@@ -4,29 +4,45 @@ class Luarocks < Formula
   sha256 "0e1ec34583e1b265e0fbafb64c8bd348705ad403fe85967fd05d3a659f74d2e5"
   head "https://github.com/keplerproject/luarocks.git"
 
+  patch :DATA
+
   depends_on "mashape/kong/openresty"
 
   def install
     openresty = Formula["mashape/kong/openresty"]
 
-    # Install to the Cellar, but the tree to install modules is in HOMEBREW_PREFIX
+    # Install to the Cellar, but the tree to install modules is in HOMEBREW_PREFIX/opt/kong
     args = [
       "--prefix=#{prefix}",
-      "--rocks-tree=#{openresty.prefix}/luajit",
+      "--rocks-tree=#{HOMEBREW_PREFIX}/opt/kong",
       "--sysconfdir=#{prefix}/etc",
       "--with-lua=#{openresty.prefix}/luajit",
       "--with-lua-include=#{openresty.prefix}/luajit/include/luajit-2.1",
       "--lua-suffix=jit"
     ]
 
+    ENV.deparallelize
```

Comment:
:+1: wasn't sure if it was related to jobs argument, but this makes sense, and yes indeed, the build process is too fast for this option to make any significant difference...